sentry/src/routes/channel.rs

```diff
@@ -169,7 +192,7 @@ pub async fn insert_events<A: Adapter + 'static>(
     let session = req
         .extensions()
         .get::<Session>()
-        .expect("request session")
+        .map(ToOwned::to_owned)
```

Comment:
You already receive a reference from `get()`, why do you need to own it and then `get_ref` again?
Just `get()` and then use it for the `record()`

Reply:
This is because we use `req.into_body` which consumes `req` and invalidates the references from `get()`